src/js/tech/html5.js

```diff
@@ -73,8 +73,10 @@ class Html5 extends Tech {
     // Our goal should be to get the custom controls on mobile solid everywhere
     // so we can remove this all together. Right now this will block custom
     // controls on touch enabled laptops like the Chrome Pixel
-    if (browser.TOUCH_ENABLED && options.nativeControlsForTouch === true) {
-      this.trigger('usenativecontrols');
+    if ((browser.TOUCH_ENABLED && options.nativeControlsForTouch === true) || browser.IS_IPHONE) {
+      this.setTimeout(function() {
```

Comment:
If just set `controls = true` here instead of the extra indirection of the event, I'm in favor.